src/arch/xtensa/boot_loader.c

```diff
@@ -26,6 +26,28 @@
 
 #if CONFIG_BOOT_LOADER
 
+/* generic string compare used by boot loader */
+int strcmp(const char *s1, const char *s2)
```

Comment:
Can we add additional comment, that we are cloning this method here in order to reduce the size of the bootloader's code?

Reply:
This does not reduce the size of boot loader , it is cloned to make code more readable.

Reply:
@dabekjakub Can you also remove bmem* functions from here? I think they are also exact copies of whats in lib.